### PR TITLE
Update Cosmos DB Management to 01-15 

### PR DESCRIPTION
**Pull request by kristynhamasaki**
# All SDK Contribution checklist:

Based on this Swagger spec version 01-15 https://github.com/Azure/azure-rest-api-specs/pull/12261

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] **Please open PR in `Draft` mode if it is:**
	- Work in progress or not intended to be merged.
	- Encountering multiple pipeline failures and working on fixes.
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/breaking-change-rules.md).**

### [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code. (Track 2 only)
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK. Please double check nuget.org current release version.

## Additional management plane SDK specific contribution checklist: 
Note: Only applies to `Microsoft.Azure.Management.[RP]` or `Azure.ResourceManager.[RP]`
 
- [ ] Include updated [management metadata](https://github.com/Azure/azure-sdk-for-net/tree/master/eng/mgmt/mgmtmetadata).
- [X] Update AzureRP.props to add/remove version info to maintain up to date API versions.

### Management plane SDK Troubleshooting
- If this is very first SDK for a services and you are adding new service folders directly under /SDK, please add `new service` label and/or contact assigned reviewer.
- If the check fails at the `Verify Code Generation` step, please ensure:
	- Do not modify any code in generated folders.
	- Do not selectively include/remove generated files in the PR.
	- Do use `generate.ps1/cmd` to generate this PR instead of calling `autorest` directly.
	Please pay attention to the @microsoft.csharp version output after running generate.ps1. If it is lower than current released version (2.3.82), please run it again as it should pull down the latest version,

### Old outstanding PR cleanup
 Please note:
	If PRs (including draft) has been out for more than 60 days and there are no responses from our query or followups, they will be closed to maintain a concise list for our reviewers.

### CODE REVIEW

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj [hunk 1]
```diff
@@ -6,12 +6,13 @@
...
     <Description>Provides developers with libraries for the CosmosDB under Azure Resource manager to manage CosmosDB Account, Databases and child resources and their available management capabilities. Create, Delete, Update CosmosDB Account, Databases and child resources. Note: This client library is for CosmosDB under Azure Resource Manager.</Description>
-    <Version>1.4.1-preview</Version>
+    <Version>2.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.CosmosDB</AssemblyName>
     <PackageTags>management;cosmosdb;</PackageTags>
     <PackageReleaseNotes>
       <![CDATA[
-        Fixing error propagation for APIs concerning Data Plane RBAC and Point in Time Restore feature.
+        Updating with the apiVersion 2020-01-15 azure-rest-api-specs.
+        This patch includes Network Acl Bypass support and Managed Service Identity.
```
Comment (MehaKaushik) [inline sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj:15]: MAke this more elaborate, cover all 4 new features being introduced.

**Review by MehaKaushik — APPROVED** [review]

File: sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj (hunk 1 above)
Comment (kristynhamasaki) [reply sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Microsoft.Azure.Management.CosmosDB.csproj]: Done, thanks!

**Comment by kristynhamasaki** [comment]
Hi @allenjzhang Please take a look at this PR when you have the chance! 

It is just failing the build check with "Discrepancy detected between generated code in PR and reference generation." in one file: [Generated/Models/BackupPolicy.cs](https://github.com/Azure/azure-sdk-for-net/blob/c1befaae93b57755dc0bd9e83638e7346f35c3a8/sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Generated/Models/BackupPolicy.cs) where my version has Newtonsoft.Json but the pipeline's version doesn't. I generated the code using generate.ps1 but I am still seeing this difference. Please let me know if I should manually make the change to the file to pass the check. Thanks!

**Review by allenjzhang — APPROVED** [review]

**Pull request merged**